server/bundles/io.cloudbeaver.model/src/io/cloudbeaver/auth/UserLoginRecord.java

```diff
@@ -14,11 +14,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.cloudbeaver.service.security.bruteforce;
+package io.cloudbeaver.auth;
 
 import org.jkiss.dbeaver.model.auth.SMAuthStatus;
 
 import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.Map;
 
-public record UserLoginRecord(SMAuthStatus smAuthStatus, LocalDateTime time) {
+public record UserLoginRecord(SMAuthStatus smAuthStatus,
+    LocalDateTime time,
+    Map<String, Object> authState
```

Comment:
Not sure that it is safe to read auth state. What changed, why do we need it now?